tests/unit-tests/performance.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Tests the Performance optmisations.
+ *
+ * @package Newspack\Tests
+ */
+
+use Newspack\Performance;
+
+/**
+ * Tests the performance optmisations.
+ */
+class Newspack_Test_Newspack_Performance extends WP_UnitTestCase {
+	/**
+	 * Test CSS minification.
+	 */
+	public function test_performance_css_minification() {
+		$html_with_style_tags = '<html>
+			<style>
+				/* Here is a comment*/
+				/**
+				 * And this is a multi-line comment, with a link: https://example.com/hello-world
+				 */
+				body {
+					color: red;
+				}
+			</style>
+		</html>';
+		self::assertEquals(
+			'<html>
+			<style>body{color: red;}</style>
```

Comment:
Definitely a nit, but maybe more savings to be had: couldn't we strip any whitespace space after the colon in `color: red;`?

Reply:
That would be possible, but I'd prefer to keep the regular expression simple and less error-prone.